config/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- 100-namespace.yaml
+- 200-clusterrole.yaml
+- 200-serviceaccount.yaml
+- 201-clusterrolebinding.yaml
+- 300-transformation.yaml
+- 500-controller.yaml
+- config-leader-election.yaml
```

Comment:
@tzununbekov should I add `config-logging.yaml` and `config-observability.yaml` here?

Reply:
yes, we need them here too